core/src/replay_stage.rs

```diff
@@ -708,8 +697,28 @@ impl ReplayStage {
             );
         }
         inc_new_counter_info!("replay_stage-tower_duration", ms as usize);
+        rv
+    }
 
-        votable
+    #[allow(clippy::type_complexity)]
```

Comment:
Such a great warning. Why disable?

Reply:
doesn't like tuples of size 4

Reply:
But your convinced the code is better with a tuple of size 4?